### ISSUE TITLE
firebase deploy: Warning: advanced compression encodings unavailable without ES6/C++11. Falling back to gzip.

### ISSUE THREAD

**Issue by mokupindia** [firebase/firebase-tools#422]
### Steps to reproduce

Installed the latest Firebase tools as below 

sudo npm install -g firebase-tools

firebase login

firebase init 

```
? Which Firebase CLI features do you want to setup for this folder? Press Space 
to select features, then Enter to confirm your choices. Hosting: Configure and d
eploy Firebase Hosting sites

=== Project Setup

First, let's associate this project directory with a Firebase project.
You can create multiple project aliases by running firebase use --add, 
but for now we'll just set up a default project.

i  .firebaserc already has a default project, skipping

=== Hosting Setup

Your public directory is the folder (relative to your project directory) that
will contain Hosting assets to be uploaded with firebase deploy. If you
have a build process for your assets, use your build's output directory.
```

When I run firebase deploy i get below error

```
Warning: advanced compression encodings unavailable without ES6/C++11. Falling back to gzip.

⚠  Your CLI authentication needs to be updated to take advantage of new features.
⚠  Please run firebase login --reauth

```

Even if I run firebase login --reauth, again same error 

PS: I had used older version of firebase tools earlier which was mapped to a different firebase project ID. i am facing this issue post update to latest firebase tools update.

If i do firebase --version i get below:
```
Warning: advanced compression encodings unavailable without ES6/C++11. Falling back to gzip.
3.9.2
```
OS : macOS Sierra
Node Version : v6.11.1
NPM : npm@3.10.10

**Comment by mokupindia** [comment]
Solution to problem 1 : You should delete firebase.json and .firebaserc if it exists in you sysment that is still refering any old firebase project.

Solution to problem 2 : ???? Warning: advanced compression encodings unavailable without ES6/C++11. Falling back to gzip.

**Comment by mbleigh** [comment]
A new version of our dependency `superstatic` was logging this warning in version 4.2.0. We've just released 4.2.1 which shouldn't log this warning anymore. Try running `npm i -g firebase-tools` again, which should solve the issue.
